Cloud/GCE/push.sh

```diff
@@ -7,5 +7,5 @@ set -e
 
 source config.sh
 
-docker tag fuzzilli gcr.io/$PROJECT_ID/$CONTAINER_NAME
-docker push gcr.io/$PROJECT_ID/$CONTAINER_NAME
+docker tag $CONTAINER_NAME:latest $REGION-docker.pkg.dev/$PROJECT_ID/fuzzilli-docker-repo/$CONTAINER_NAME
```

Comment:
I'd prefer if changes to these files could be a separate CL (if you want to include them) :)

Reply:
I'll revert this change it is not relevant to the PR. :)